sdk/cosmos/azure-cosmos/doc/setup.md

```diff
@@ -0,0 +1,137 @@
+This document currently details Python commands for building Azure Cosmos SDK.
+
+### Fork and clone the repository
+To build and develop locally, it is strongly recommended to fork and clone the repository: https://github.com/Azure/azure-sdk-for-python
```

Comment:
I would suggest removing strongly recommended and just directly instructing to fork and clone the repo.

Reply:
Done